package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "branch-sdk",
-  "version": "2.85.1",
+  "version": "0.0.0",
```

Comment:
what's the point of zeroing it out?

Reply:
since we are no longer going to be checking in updates to this file, it makes it clear that the version numbers that we publish come from somewhere else